src/main/frontend/dicts.cljs

```diff
@@ -390,7 +390,8 @@
 
         :tips/all-done "All Done"
 
-        :command-palette/prompt "Type a command"}
+        :command-palette/prompt "Type a command"
+        :select/prompt "Select one"}
```

Comment:
Better use "select a graph"

Reply:
The component is meant to be generic/reusable which is why the wording isn't specific

Reply:
We could. I can add support for an optional :prompt-key in config for the graph wording and default to "Select one" for those that don't need specific language